doc/nrf/app_dev/device_guides/nrf54l/building_nrf54l.rst

```diff
@@ -0,0 +1,108 @@
+.. _building_nrf54l:
+
+Building and programming with nRF54L15 DK
+#########################################
+
+This guide provides instructions on how to build and program the nRF54L15 development kit.
+Whether you are working with single or multi-image builds, the following sections will guide you through the necessary steps.
+
+Depending on the sample, you must program only the application core or both the VPR and the application core.
+Additionally, the process will differ based on whether you are working with a single-image or multi-image build.
+
+Using the command line
+======================
+
+To build samples supporting the nRF54L15 DK from the command line, use :ref:`west <zephyr:west>`.
```

Comment:
Not sure how to organize this section, there are a few cases here:
- Building for APP only
- Building for both APP and FLPR
  - In separate builds
  - With sysbuild
     - Using VPR launcher
     - Using application that supports multi-image

Reply:
I tried to incorporate the new structure, please double-check